core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -93,15 +95,6 @@
     height: 2.5em;
   }
 
-  topics-list .delete-icon {
-    border-bottom: 1px solid #fff;
-    padding-left: 0.8vw;
-  }
-
-  topics-list .list-item:hover .delete-icon {
```

Comment:
Why removing this effect?

(Can you please share a screenshot of the new UI? Make sure that the new UI is similar to the existing UI.)

Reply:
Sorry that was a mistake, I have brought it back. I have also brought back the delete-icon class for this file, because it had some style attributes associated with it which were required.
Also, delete-icon class is required to be associated with <td> tag, because on hover, it is supposed to stay white.
Screenshots:
Before:
![Screenshot from 2020-04-14 20-55-41](https://user-images.githubusercontent.com/42586894/79243279-0eb11500-7e93-11ea-9087-5ffcdc687460.png)
After:
![Screenshot from 2020-04-14 20-57-41](https://user-images.githubusercontent.com/42586894/79243297-14a6f600-7e93-11ea-93b0-d33487d1fa59.png)

Reply:
Thanks for sharing the screenshot, LGTM! :)